ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParsePayload.java

```diff
@@ -235,7 +235,8 @@ long calculateSampleId(String clientId) {
     return crc32.getValue() % 100;
   }
 
-  private static String normalizeUuid(String v) {
+  @VisibleForTesting
+  static String normalizeUuid(String v) {
```

Comment:
I made this public to be able to write unit tests for it. But I'm not sure if there might be a good reason for it to stay `private`

Reply:
I'm fine with this. You've marked it `@VisibleForTesting` which is perfectly in line with established practice in this codebase.